Allow path changes within `SaveBoundary` (e.g., `RouterTabs`) by automatically creating a `SubRoute` (#1795)
vivid-planet/comet: .changeset/brown-kids-develop.md

---
"@comet/admin": major
---

> SaveBoundary now creates a subroute by default
> 
> default path is ./save, change it using subRoutePath prop

Reviewer: ```suggestion
Create a subroute by default in `SaveBoundary`

The default path is `./save`, you can change it using the `subRoutePath` prop.
```
